tests/foreman/cli/test_computeresource_rhev.py

```diff
@@ -380,8 +380,10 @@ def test_negative_add_image_rhev_with_invalid_name(rhev, module_os):
 @pytest.mark.on_premises_provisioning
 @pytest.mark.tier3
 @pytest.mark.rhel_ver_match('[^6]')
+@pytest.mark.parametrize('setting_update', ['destroy_vm_on_host_delete'], indirect=True)
 def test_positive_provision_rhev_with_host_group(
     request,
+    setting_update,
```

Comment:
Do you set the value of the this settings anywhere lower in test, fixture or elsewhere? I just can't see the purpose.

Reply:
Good catch @vsedmik, it was just a copy paste for indirect parametrization we've been using for all tests, I've updated the setting_update  fixture to accept setting name and value through request.param, and it works I tested it locally.
@vsedmik @jyejare Do you mind re-reviewing this PR, Thank you! ❤️

Reply:
I mean the initial question, do we really need this parameterization, Because by default this setting is `True`.

Do you see any test is changing this to False making VMs dropped?

Reply:
yeah, we need it, default value for this setting is False which doesn't delete host on CR, and we changing it to True to make sure provisioned host is cleaned up from CR